demo/image-segmentation/pytorch_loader.py

```diff
@@ -233,42 +238,55 @@ def __init__(
             max_parallelism=self.config.num_processes,
             project=self.project_name,
             bucket=self.bucket_name,
-            sort_results=self.config.
-            sort_listing_results,  # This needs to be True to map images with labels.
+            # This needs to be True to map images with labels.
+            sort_results=self.config.sort_listing_results,
             prefix=images_prefix,
         ).run()
         self.labels = dataflux_core.fast_list.ListingController(
             max_parallelism=self.config.num_processes,
             project=self.project_name,
             bucket=self.bucket_name,
-            sort_results=self.config.
-            sort_listing_results,  # This needs to be True to map images with labels.
+            # This needs to be True to map images with labels.
+            sort_results=self.config.sort_listing_results,
             prefix=labels_prefix,
         ).run()
 
     def __len__(self):
         return len(self.images)
 
     def __getitem__(self, idx):
-        image = np.load(
-            io.BytesIO(
-                dataflux_core.download.download_single(
-                    storage_client=self.storage_client,
-                    bucket_name=self.bucket_name,
-                    object_name=self.images[idx][0],
-                )), )
-
-        label = np.load(
-            io.BytesIO(
-                dataflux_core.download.download_single(
-                    storage_client=self.storage_client,
-                    bucket_name=self.bucket_name,
-                    object_name=self.labels[idx][0],
-                )), )
+        try:
+            image = np.load(
+                io.BytesIO(
+                    dataflux_core.download.download_single(
+                        storage_client=self.storage_client,
+                        bucket_name=self.bucket_name,
+                        object_name=self.images[idx][0],
+                    )
+                ),
+            )
+        except Exception as err:
+            print(f"Error trying to access {self.images[i][0]}: {err}")
+            image = None
+
+        try:
+            label = np.load(
+                io.BytesIO(
+                    dataflux_core.download.download_single(
+                        storage_client=self.storage_client,
+                        bucket_name=self.bucket_name,
+                        object_name=self.labels[idx][0],
+                    )
+                ),
+            )
+        except Exception as err:
```

Comment:
Similar question as above, does it make sense to combine these try/except blocks?